src/select/index.js

```diff
@@ -32,6 +32,35 @@ class Select extends Component {
     });
   };
 
+  handleKeyDown = ({ key }) => {
+    const { focus } = this.state;
+    const { options } = this.props;
+    switch (key) {
+      case 'ArrowDown':
+        this.setState(prevState => ({
+          focus: (
+            (prevState.focus === undefined
+              ? -1
+              : prevState.focus
+            ) + 1
+          ) % (options.length),
+        }));
+        break;
+      case 'ArrowUp':
```

Comment:
This looks fine for handling through keyboard events. In the playground, on using ArrowUp the window scrolls up to. Probably should we prevent the event to bubble up till window by this component itself ?

Reply:
We need to restrict this though before merging or find if there is a way to block this effect.

Reply:
Wasn't able to reproduce the same effect here on mozilla and chrome. Anyways added stop propagation to prevent the event from bubbling up to parent component. It should be fixed now.

Reply:
cool. Will test while reviewing.